README.md

```diff
@@ -9,7 +9,7 @@ The boards can be assembled by hand or purchased preassembled; the open-source
 IDE can be downloaded for free at https://www.arduino.cc/en/Main/Software
 
 * For more information, see the website at: https://www.arduino.cc/
-or the forums at: https://forum.arduino.cc/  
```

Comment:
This trailing whitespace was intentional.

Reply:
@per1234 Sorry, I missed your comment!

Unfortunately, it seems that adding a backslash here makes GitHub render it as part of the URL.